graphql/src/store/query.rs

```diff
@@ -100,11 +100,18 @@ fn list_values(value: Value) -> Vec<Value> {
 }
 
 /// Parses GraphQL arguments into an attribute name to order by, if present.
-fn build_order_by(arguments: &HashMap<&q::Name, q::Value>) -> Option<String> {
+fn build_order_by(
+    entity: &s::ObjectType,
+    arguments: &HashMap<&q::Name, q::Value>,
+) -> Option<(String, ValueType)> {
     arguments
         .get(&"orderBy".to_string())
         .and_then(|value| match value {
-            q::Value::Enum(name) => Some(name.to_owned()),
+            q::Value::Enum(name) => {
+                let field = sast::get_field_type(entity, &name)
+                    .expect("order by attribute does not belong to entity");
```

Comment:
Same here, this error needs to be returned. It also needs to be more specific and mention the types/names involved. Like:
```
Value "foo" is not valid for MyEntity_OrderBy. Valid values are: baz, ruux.
```

Our `Strings` type can be used to represent the valid values.